### PR TITLE
Hide group when empty

### PR DESCRIPTION
**Pull request by benrudolph**
http://manage.dimagi.com/default.asp?149629#832915

this code could use a serious refactor

@millerdev

### CODE REVIEW

**Comment by benrudolph** [comment]
Also, just realized the commit name is really misleading, it should be: Hide group when empty

**Comment by czue** [comment]
not an xforgasmic first impression?

**Comment by benrudolph** [comment]
sadly that was a bit of a misnomer

File: touchforms/formplayer/static/formplayer/script/fullform-ui.js [hunk 1]
```diff
@@ -61,12 +61,17 @@ function ixInfo(o) {
 }
 
 function empty_check(o, anim_speed) {
-  if (o.type == 'repeat-juncture' || o.type == 'sub-group') {
-    var empty = (o.children.length == 0);
+  if (o.type == 'repeat-juncture') {
```
Comment (millerdev) [inline touchforms/formplayer/static/formplayer/script/fullform-ui.js]: I realize you didn't change this, but it should be using `===`. Also below.

**Comment by snopoke** [comment]
lol

**Pull request merged**